api-participant/src/main/java/bio/terra/pearl/api/participant/controller/PortalController.java

```diff
@@ -46,4 +50,41 @@ public ResponseEntity<Object> listPortalEnvAlerts(String portalShortcode, String
     return ResponseEntity.ok(
         portalDashboardConfigService.findByPortalEnvId(portalEnvironment.getId()));
   }
+
+  @Override
+  @CrossOrigin(
+      origins = {
+        "https://juniperdemodev.b2clogin.com", // Heart (demo only)
+        "https://junipercmidemo.b2clogin.com", // CMI (demo only)
+        "https://ourhealthdev.b2clogin.com", // OurHealth (demo)
+        "https://ourhealthstudy.b2clogin.com", // OurHealth (prod)
+        "https://hearthivedev.b2clogin.com", // HeartHive (demo)
+        "https://hearthive.b2clogin.com", // HeartHive (prod)
+        "https://gvascdev.b2clogin.com", // gVASC (demo)
+        "https://gvascprod.b2clogin.com" // gVASC (prod)
+      },
+      maxAge = 3600,
+      methods = {RequestMethod.GET, RequestMethod.OPTIONS})
+  public ResponseEntity<Object> getBranding(
+      String portalShortcode, String envName, String language) {
+    EnvironmentName environmentName = EnvironmentName.valueOfCaseInsensitive(envName);
+
+    PortalEnvironment portalEnv =
+        portalEnvironmentService
+            .loadWithParticipantSiteContent(portalShortcode, environmentName, language)
+            .orElseThrow(() -> new NotFoundException("Portal environment not found"));
+
+    LocalizedSiteContent siteContent =
+        portalEnv.getSiteContent().getLocalizedSiteContents().stream()
+            .findFirst()
+            .orElseThrow(() -> new NotFoundException("Site content not found"));
+
```

Comment:
Let's make this branding object a DTO class so that it can be constructed with a more readable builder